libs/librepcb/library/pkg/packagecheck.cpp

```diff
@@ -154,8 +154,9 @@ void PackageCheck::checkPadsOverlapWithPlacement(MsgList& msgs) const {
       std::shared_ptr<const FootprintPad> pad = it.ptr();
       std::shared_ptr<const PackagePad>   pkgPad =
           mPackage.getPads().find(pad->getUuid());
-      Length clearance(150000);  // 150um
-      Path   stopMaskPath = pad->getOutline(clearance);
+      Length clearance(150000);  // 150 µm
+      Length tolerance(000100); // 0.1 µm, to avoid rounding issues
```

Comment:
I guess this is not 0.1µm because literals starting with `0` are interpreted as octal numbers ;) And shouldn't 0.01µm be enough?

Reply:
D'oh :smile: Will fix.

Reply:
Done.